bucket_map/src/index_entry.rs

```diff
@@ -14,7 +14,17 @@ use {
 
 /// allocated in `contents` in a BucketStorage
 pub struct BucketWithBitVec {
-    pub occupied: BitVec,
+    occupied: BitVec,
+    capacity_pow2: Capacity,
```

Comment:
It could, but soon I will distinguish them as this one has to be pow2 because it will only be data bucket. So, to make it clear what it needed to represent, I decided to leave it named the same.